addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml

```diff
@@ -171,6 +171,15 @@
 				<option value="12">MULTI CH IN</option>
 				<option value="33">ADAPTER PORT</option>
 				<option value="27">SIRIUS</option>
+				<option value="38">Internet Radio</option>
```

Comment:
I do see that all the other are spelled with caps, I also would expect `Line 1` or `LINE 1` instead of `Line1` @Stratehm wdyt?

Reply:
I wrote it like it shows in display of AVR